### ISSUE TITLE
Add `presentation::make_semigroup_presentation`

### ISSUE THREAD

**Issue by james-d-mitchell** [libsemigroups/libsemigroups#438]
Basically just wrapping:

```
presentation::replace_word(p, {}, {p.alphabet().size()});
presentation::add_identity_rules(p, p.alphabet().size());
p.alphabet(p.alphabet().size() + 1);
```

or equivalent.

**Comment by MTWhyte** [comment]
I will do this

**Comment by james-d-mitchell** [comment]
Resolved in v2.6.0
